libpromises/evalfunction.c

```diff
@@ -8879,6 +8884,9 @@ void ModuleProtocol(EvalContext *ctx, const char *command, const char *line, int
 
             StringSetAddSplit(tags, content, ',');
             StringSetAdd(tags, xstrdup("source=module"));
```

Comment:
We can drop the above line now, right? I know it's adding to a set so the resulting set will be the same, but it's extra allocations and deallocations for no reason.

Reply:
I'm not sure I understand. A few lines above, the string set is cleared. Thus, we need to add the "default" tags again.

Reply:
I didn't see the `StringSetClear()` call in the diff. Sorry for the noise.